map_server/src/main.cpp

```diff
@@ -91,25 +91,25 @@ class MapServer
 #endif
         try {
           doc["resolution"] >> res;
-        } catch (YAML::InvalidScalar) {
+        } catch (YAML::InvalidScalar &) {
```

Comment:
These should be submitted upstream rather than here isn't it ? Or is it ROS 2 specific code?